functions.php

```diff
@@ -42,6 +42,7 @@ function( $template ) {
 		'P4_Campaigns',
 		'P4_Post_Campaign',
 		'P4_Campaign_Exporter',
+		'P4_Campaign_Importer',
```

Comment:
Do we have to load this always? This way it will always create this object. Can we instantiate this only when needed?

Reply:
We should do this also for other classes as well (P4_Post_Report_Controller, P4_Control_Panel, etc) and make sure they load only in wp admin section and if possible only in the specific pages that are actually needed.

I will create a ticket again for this (since this is still relevant after all).